content/docs/guides/manage-database-access.md

```diff
@@ -0,0 +1,108 @@
+
+Your default Neon role is automatically granted membership in a `neon_superuser` role, which provides the user with the privileges and predefined PostgreSQL role memberships shown in this `CREATE ROLE` statement:
+
+<CodeBlock shouldWrap>
+
+```sql
+CREATE ROLE neon_superuser CREATEDB CREATEROLE NOLOGIN IN ROLE pg_read_all_data, pg_write_all_data;
+```
+
+</CodeBlock>
+
+You can think of this role as a Neon administrator role. A user with membership in the `neon_superuser` role can create databases, create roles, add extensions, grant `neon_superuser` privileges, and has all the privileges of `pg_read_all_data` and `pg_write_all_data`. You can find more information about this role [here](/docs/reference/manage/roles#the-neon-super-user).
+
+Any user created in the Neon console or using the Neon API is automatically granted membership in the `neon_superuser` role. But what do you do if you need to create roles with different or limited privileges? After all, not every database user should be an administrator in Neon.
+
+Neon supports creating and managing PostgreSQL roles with SQL. Roles created with SQL from a client such as [psql](/docs/connect/query-with-psql-editor) or from the [Neon SQL Editor](/docs/get-started-with-neon/query-with-neon-sql-editor) start with the same basic privileges granted to newly created roles in stand-alone PostgreSQL. They are not granted membership in the `neon_superuser` role.
+
+Using SQL, you can define database roles with only the privileges you choose to grant. The following instructions show you how.
+
+## Create roles and grant database privileges
+
+To begin, assume you're creating a new database that will be used by several developers, all requiring read-write access.
+
+1. Start by connecting to the default `neondb` database with your default Neon role using `psql` (or some other client that supports SQL). As described above, this role has administration privileges in Neon, which enable it to create databases and roles.
+
+    <CodeBlock shouldWrap>
+
+    ```bash
+    psql postgres://john:<password>@ep-restless-waterfall-733645.us-west-2.aws.neon.tech/neondb
+    ```
+
+    </CodeBlock>
+
+2. Create a new database. Call it `app_db`. Neon supports creating databases with the Neon console, the Neon API, and SQL. Here, we use SQL.
+
+    ```sql
+    CREATE DATABASE app_db;
+    ```
+
+    <Admonition type="note">
+    The role that creates a database is automatically the owner of the database.
+    </Admonition>
+
+3. Create a shared role for database users. This role will be used to manage database user privileges. Neon requires a password when creating any role with SQL.
+
+    ```sql
+    CREATE ROLE dev_users PASSWORD `password`;
```

Comment:
it is worth mentioning that we do not support creating users with hashed passwords, like

```sql
CREATE USER user_name
WITH PASSWORD 'SCRAM-SHA-256$4096:UunGvPETiX/JNGBvjOgW9A==$CPGNh7/MRfs0ispH9/HSJajOI8Uhp+UCRo/b/ToXIEY=:L6NzxQ3XUeWEeRa+oiuajC9Vgl7wk6ZpHAHl+pv4m00=';
```

Reply:
Added "Hashed passwords are not supported" and that passwords are required.